pkgs/top-level/all-packages.nix

```diff
@@ -12291,6 +12291,11 @@ let
 
   koji = callPackage ../tools/package-management/koji { };
 
+  konversation_1_6 = callPackage ../applications/networking/irc/konversation/1.6.nix {
```

Comment:
Please add this to [`kdePackagesFun`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L14763) as `konversation` instead.